testutils/src/main/java/com/microsoft/identity/internal/testutils/labutils/ConfidentialClientHelper.java

```diff
@@ -0,0 +1,63 @@
+package com.microsoft.identity.internal.testutils.labutils;
+
+import com.microsoft.identity.common.exception.ClientException;
+import com.microsoft.identity.common.internal.authorities.AccountsInOneOrganization;
+import com.microsoft.identity.common.internal.authorities.AzureActiveDirectoryAuthority;
+import com.microsoft.identity.common.internal.providers.oauth2.OAuth2Strategy;
+import com.microsoft.identity.common.internal.providers.oauth2.TokenRequest;
+import com.microsoft.identity.common.internal.providers.oauth2.TokenResult;
+
+import java.io.IOException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+import java.security.UnrecoverableKeyException;
+import java.security.cert.CertificateException;
+
+abstract class ConfidentialClientHelper {
+
+    private final static String GRANT_TYPE = "client_credentials";
+    private final static String TENANT_ID = "72f988bf-86f1-41af-91ab-2d7cd011db47";
```

Comment:
nit: add a comment specifying whose tenantID this is.

Reply:
Addressed